pkg/scannerv4/constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+const (
+	// FixedByName is the name of the FixedBy enricher
+	FixedByName = "fixedby"
+	// FixedByType is the type of data returned from the FixedBy Enricher's Enrich method.
+	FixedByType = "message/vnd.stackrox.scannerv4.fixedby; enricher=" + FixedByName
+
+	// NVDName is the name of the NVD enricher
+	NVDName = `nvd`
+	// NVDType is the type of data returned from the NVD Enricher's Enrich method.
+	NVDType = `message/vnd.stackrox.scannerv4.vulnerability; enricher=` + NVDName + ` schema=https://csrc.nist.gov/schema/nvd/api/2.0/source_api_json_2.0.schema`
+
+	// ManualUpdaterName is the name of the manual updater
+	ManualUpdaterName = "stackrox-manual"
+)
```

Comment:
Would that be also a good place for the following strings from one of the other files in this PR?

```
		API:                "https://catalog.redhat.com/api/containers/",
		Repo2CPEMappingURL: "https://access.redhat.com/security/data/metrics/repository-to-cpe.json",
		MediaType: "application/vnd.claircore.filesystem",
```

Reply:
Let's leave these for now - I have a follow up item planned (ROX-25698) to refactor the compliance package to better reflect the new architecture. I'd vote for moving these in that context.